cime_config/testdefs/testmods_dirs/clm/Fates/shell_commands

```diff
@@ -1,2 +1,16 @@
 ./xmlchange CLM_BLDNML_OPTS="-no-megan" --append
 ./xmlchange BFBFLAG="TRUE"
+
+CASE=`./xmlquery CASE --value`
+SRCROOT=`./xmlquery SRCROOT --value`
+
+#CIME_OUTPUT_ROOT=`./xmlquery CIME_OUTPUT_ROOT --value`
```

Comment:
I'm not sure why this commented out code is here? Does this just document some things you tried that didn't seem to work?

It's also puzzling to me that these things wouldn't work. I'm guessing it's just because of where in the process shell_commands gets invoked. It must just be before some of the test infrastructure is added...

Reply:
Comment out code was removed so resolving.